client/rhel/spacewalk-client-tools/src/bin/rhn_check.py

```diff
@@ -359,7 +358,7 @@ def __build_status_report():
                 pass
 
         # We need to fit into xmlrpc's integer limits
-        if status_report['uptime'][1] > long(2)**31-1:
+        if status_report['uptime'][1] > int(2)**31-1:
```

Comment:
What is this good for in the first place? Just for fun I tried this with both python2 *AND* python3 and it works in both versions:

#!/usr/bin/python2

foo=4200000000000000

if foo > 2**31-1:
    print ("too big")
else:
    print ("ok")

I'm a C guy, but there this would be a case that only affects the number "2", so to me this looks completely superfluous.

Nevertheless I will approve as it works with both python versions.